docs/developer-guide.md

```diff
@@ -1,4 +1,4 @@
-#WIP: Developer guide
+# WIP: Developer guide
```

Comment:
It would be more appropriate to have WIP as 'tag' (Some thing like

> Work-In-Progress

instead of in the heading).

Reply:
Done https://github.com/gluster/gdeploy/pull/536